### PR TITLE
Addhigh chart

### PR DESCRIPTION
**Pull request by mugijanvier**
@alptilev

### CODE REVIEW

**Comment by mugijanvier** [comment]
check its!!

File: sesh/settings.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 TEMPLATE_DIR = os.path.join(BASE_DIR,'templates')
 
 config = RawConfigParser()
-config.read( os.path.join(BASE_DIR,'settings_local.ini'))
+config.read( os.path.join(BASE_DIR, "ocal_settings.ini"))
```
Comment (jlark) [inline sesh/settings.py]: Please use the correct name for this file.

It settings_local.ini

File: seshdash/api/wunderground.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import requests json
```
Comment (jlark) [inline seshdash/api/wunderground.py]: Why was request removed?

File: seshdash/static/bower_components/bootstrap/dist/js/bootstrap.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /*!
```
Comment (jlark) [inline seshdash/static/bower_components/bootstrap/dist/js/bootstrap.js]: Please don't check in Anything inside bower_components.

**Comment by jlark** [comment]
Hi Janvier,
I still see a bunch of files being included from bower_components. git should ignore these files. With 300 files getting checked in I can't see the code that you wrote. Please correct this.

Alp

**Pull request closed without merging**